knowledge-base/kb-listbox-scroll-per-pixel.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Scroll Per Pixel
```

Comment:
Add the name of the control also in the title. You can use only ListBox, without Rad.